datasets/snli/snli.md

```diff
@@ -0,0 +1,116 @@
+
+### Normalization information
+
+The premises from the Flickr 30k corpus corrected for spelling using the Linux spell checker and ungrammatical sentences were removed. Bowman et al. do not report any normalization, though they note that punctuation and capitalization are often omitted. 
+
+### Annotation process
+
+56,941 of the total sentence pairs were further annotated in a validation task. Four annotators each labeled a premise-hypothesis pair for entailment, contradiction, or neither, resulting in 5 total judgements including the original hypothesis author judgement. See Section 2.2 for more details (Bowman et al., 2015). 
+
+The authors report 3/5 annotator agreement on 98% of the validation set and unanimous annotator agreement on 58.3% of the validation set. If a label was chosen by three annotators, that label was made the gold label. Following from this, 2% of the data did not have a consensus label and was labeled '-' by the authors. 
+
+Label | Fleiss κ
+------|---------
+_contradiction_ | 0.77
+_entailment_ | 0.72
+_neutral_ | 0.60
+overall | 0.70
+
+## Dataset Structure
+
+### Splits, features, and labels
+
+The SNLI dataset has 3 splits: _train_, _validation_, and _test_. 
+Dataset Split | Number of Instances in Split
+--------------|--------------------------------------------
+Train | 550,152
+Validation | 10,000
+Test | 10,000
+
+Each data instance contains the following features: _premise_, _hypothesis_, _label_.
+
+Feature | Mean Token Count
+--------|-----------------
+Premise | 14.1
+Hypothesis | 8.3
+
+The _label_ has 4 possible values, _0_, _1_, _2_, _-_.  which correspond to _entailment_, _neutral_, _contradiction_, and _no label_ respectively. The dataset was developed so that the first three values would be evenly distributed across the splits. See the Annotation Process section for details on _no label_.
+
+### Span indices
+
+No span indices are included in this dataset.
+
+### Example ID
+
+The ID is an integer starting from 0. It has no inherent meaning. 
+
+### Free text description for context (e.g. describe difference between title / selftext / body in Reddit data) and example
+
+For each ID, there is a string for the premise, a string for the hypothesis, and an integer for the label. Note that each premise appears three times with a different hypothesis and label. See the [SNLI corpus viewer](https://huggingface.co/nlp/viewer/?dataset=snli) to explore more examples.
+
+ID | Premise | Hypothesis | Label
+---|---------|------------|-------
+0	| Two women are embracing while holding to go packages.	| The sisters are hugging goodbye while holding to go packages after just eating lunch.	| 1
+1	| Two women are embracing while holding to go packages.	| Two woman are holding packages.	| 0
+2	| Two women are embracing while holding to go packages.	| The men are fighting outside a deli. | 2
+
+## Tasks supported:
```

Comment:
I feel this section should be presented before as it's probably the information a user wants to know first.

Reply:
Agreed. It's also short enough that it doesn't push the author information too far down/

Reply:
I agree !

Furthermore it'd be cool to have a section in yaml with the tasks. It would allow moon-landing to parse it and allow users to do dataset search by task :)